haas/cli.py

```diff
@@ -136,6 +131,8 @@ def serve_networks():
 def init_db():
     """Initialize the database"""
     from haas import model
+    server.register_drivers()
```

Comment:
Should we just call server.api_server_init() and add a `stop_consoles=True` default which gets overridden here?